src/pages/LandingPage/sections/SectionThree/FormComponents/ContactForm.js

```diff
@@ -13,19 +13,87 @@ TODO: - Responsive to screen size.
  ************************/
 
 import React, { Component } from 'react';
+import * as emailjs from 'emailjs-com';
 
 import classes from './ContactForm.module.scss';
 import Button from '../../../../../components/Button/Button';
+import Span from '../../../../../components/Span/Span';
 
 class ContactForm extends Component {
     constructor() {
         super();
-        this.state = {};
+        this.state = {
+            name: '',
+            email: '',
+            message: '',
+            span: null
+        };
     }
 
+    handleSubmit(e) {
+        e.preventDefault();
+        const { name, email, message } = this.state;
+        this.setState({
+            span: <img src={require('../../../../../assets/loading.svg')} />
+        });
+
+        //add this simple check for begging -
+        if (name.length > 0 && email.length > 0 && message.length > 0) {
+            let templateParams = {
+                name: name,
+                email: email,
+                message: message
+            };
+
+            emailjs
+                .send(
+                    // 'default_service' and 'visual-music' are coresponse to visual-music user at emailjs.com
+                    'default_service',
+                    'visual_music',
+                    templateParams,
+                    process.env.REACT_APP_emailJS
+                )
+                .then(response => {
+                    this.setState({
+                        span: null
+                    });
+                    alert('email sent succsefully');
+                })
+                .catch(err => {
+                    console.log('FAILED', err);
+                    this.setState({
+                        span: err
+                    });
+                });
+        } else {
+            this.setState({ span: 'unsucssesful attempt' });
```

Comment:
Grammar mistake, as well as add some background, to have contrast against landing page. Also, add some padding-top, so that it won't be so close to the button.